ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -5,41 +5,47 @@
   @showEditToggle={{if @model.isNew false true}}
   as |form|
 >
-  <form.input
-    @name='name'
-    @type='text'
+  <Hds::Form::TextInput::Field
+    @isRequired={{true}}
+    @isOptional={{false}}
```

Comment:
Are we marking all non-required fields as `optional` as well from here on out?

Reply:
I thought that was part of the HDS migration. We would start marking all fields with required or optional. @randallmorey and @DhariniJeeva would you provide some input on this topic please?